mkdocs.yml

```diff
@@ -127,10 +127,10 @@ nav:
       - Create a Backstage Plugin: 'plugins/create-a-plugin.md'
       - Plugin Development: 'plugins/plugin-development.md'
       - Structure of a plugin: 'plugins/structure-of-a-plugin.md'
-      - Plugin Development: 'plugins/plugin-development.md'
```

Comment:
Should this really be deleted?

Reply:
If you look at line 128, this line is a duplicate, the duplicate is ignored when built into a menu however thought it best to tidy up. Hope that is ok.

Reply:
Thanks for clarifying 👍